src/FeedIo/Standard/Rss.php

```diff
@@ -106,6 +107,7 @@ public function buildItemRuleSet() : RuleSet
         $ruleSet
             ->add(new Author(), ['dc:creator'])
             ->add(new PublicId())
+            ->add($this->getModifiedSinceRule(static::DATE_NODE_TAGNAME), ['lastBuildDate', 'lastPubDate'])
```

Comment:
`lastBuildDate` or `lastPubDate` are not available for items.

Reply:
They're not supposed to be, however I have already seen feeds with one of them in the Item node.